.es.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./.babelrc",
+  "ignore": [
+    "**/*.test.ts",
+    "**/*.test.tsx",
+    "**/*.mdx"
+  ],
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": false,
+        "modules": false
```

Comment:
what's this? and why a specific file?

Reply:
These settings are meant for the esm build.